scripts/create_lightning_rc.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+OLDVER=0.37.0
+LVER=0.38.0
```

Comment:
Should we update the version now?
```suggestion
OLDVER=0.38.0
LVER=0.39.0
```

Reply:
This script isn't meant to be used as is. Let's change this when it is used as such.